solr/core/src/java/org/apache/solr/cli/ToolBase.java

```diff
@@ -34,18 +34,29 @@ protected ToolBase(PrintStream stdout) {
   }
 
   protected void echoIfVerbose(final String msg, CommandLine cli) {
-    if (cli.hasOption(SolrCLI.OPTION_VERBOSE.getOpt())) {
+    if (verbose) {
       echo(msg);
     }
   }
 
+  /**
+   * Is this tool being run in a verbose mode?
+   *
+   * @return verbose or not
```

Comment:
Unnecessary `@return` doc as the method name clearly indicates what is returned? :)

Reply:
sure...  my javadoc looks pretty skinny with no `@return`